cmake/Platform/Windows/Packaging/Tests/conftest.py

```diff
@@ -0,0 +1,146 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+
+Pytest test configuration file.
+
+"""
+import pytest
+import boto3
+import shutil
+import os
+import io
+import threading
+import urllib.parse
+import urllib.request
+from pathlib import Path, PurePath
+from subprocess import run, list2cmdline
+from tempfile import NamedTemporaryFile, mkdtemp
+from time import sleep
+
+
+def pytest_addoption(parser):
+    """ Add custom PyTest arguments. """
+    parser.addoption("--installer-uri", action="store", default="")
+    parser.addoption("--install-root", action="store", default=PurePath('C:/O3DE/0.0.0.0'))
+    # we use a directory outside of %TEMP% because otherwise MSB8029 warns it might cause incremental build issues 
+    parser.addoption("--project-path", action="store", default=PurePath('C:/Workspace/TestProject'))
+
+
+class SessionContext:
+    """ Holder for test session constants and helper functions. """
+    def __init__(self, request):
+        # setup to capture subprocess output
+        self.log_file = request.config.getoption("--log-file")
+        self.temp_dir = mkdtemp()
```

Comment:
pytest supports a [tmpdir parameter](https://docs.pytest.org/en/6.2.x/tmpdir.html#the-tmpdir-fixture) that returns a temporary directory for storing test artifacts